src/amo/components/Addon/index.js

```diff
@@ -514,8 +526,22 @@ export class AddonBase extends React.Component {
                   <p className="Addon-summary" {...summaryProps} />
                 ) : null}
 
-                {addon &&
-                  isFireFox && (
+                {showInstallButton &&
+                  config.get('enableNewInstallButton') && (
+                    <NewInstallButton
+                      addon={addon}
+                      defaultInstallSource={defaultInstallSource}
+                      disabled={!isCompatible}
+                      enable={enable}
+                      hasAddonManager={hasAddonManager}
```

Comment:
this should always be true pretty much right? is there a possibility this could be false? If there is that case, I am wondering if we should wrap this button with this conditional check? b/c if it's it false, it won't work anyway, right?

Reply:
It can be `false`, see: https://github.com/mozilla/addons-frontend/issues/3954#issuecomment-410210610. It works without the manager, but you don't have a status and therefore it is almost like the current install button.

Reply:
sorry question this was meant for disco where I saw I nothing happening (it didn't install).

Reply:
basically on disco - what I see is that the status shows 'uninstalled' (vs unknown) which means the button doesn't get disabled and this isn't covered in the button clicks options. let me know if you see something different tho..

Reply:
I am not sure I understand.

Reply:
sorry for not being clear.  Basically tho if you hasMoz... is false, you can't install anything on disco. This seems like something we should fix, right? I thought initially the buttons might be disabled in this case but they are actually still active and then it doesn't hit any case here so clicking the button doesn't do anything:

```javascript
if (!buttonIsDisabled) {
      if ([ENABLED, INSTALLED].includes(status)) {
        buttonProps.buttonType = 'neutral';
        buttonProps.onClick = this.uninstallAddon;
        buttonProps.className = makeClassName(
          buttonProps.className,
          'NewInstallButton-button--uninstall',
        );
      } else if (status === DISABLED) {
        buttonProps.buttonType = 'neutral';
        buttonProps.onClick = this.enableAddon;
        buttonProps.className = makeClassName(
          buttonProps.className,
          'NewInstallButton-button--enable',
        );
      } else if (addon.type === ADDON_TYPE_THEME) {
        buttonProps['data-browsertheme'] = JSON.stringify(getThemeData(addon));
        buttonProps.onClick = this.installTheme;
      } else if (hasAddonManager) {
        buttonProps.onClick =
          addon.type === ADDON_TYPE_OPENSEARCH
            ? this.installOpenSearch
            : this.installExtension;
      } else if (status === UNINSTALLED) {
        buttonProps.onClick = this.installExtension;
      }
    }
```

does this help clarify?

Reply:
for me it does work, I can install themes and extensions, without the `mozAddonManager`. I have fixed this in the meantime.

Reply:
oh? ok maybe it's just an issue on my end then? I have the ff build but it actually might be a bit outdated and related to that:/

Reply:
You have to make sure all the config prefs are correctly set: https://addons-frontend.readthedocs.io/en/latest/moz-addon-manager/